Makefile

```diff
@@ -42,9 +42,16 @@ ifeq ($(OS),linux)
 
 LDLIBS=\
 	-lrt\
+	-lsystemd\
```

Comment:
Could you please make systemd as a optional dependency?

Reply:
yes